installer/src/supported-boards.js

```diff
@@ -0,0 +1,10 @@
+export default [
+    {
+        fqbn: 'arduino:mbed_nano:nano33ble',
+        platform: 'arduino:mbed_nano',
```

Comment:
I think that the core is always the lower part of the fqbn, so we could just do a substring instead of keeping them explicitly.